e2e/cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -231,7 +231,6 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
 
         // * focus should be on the input box
         cy.get('#post_textbox').should('be.focused');
-        cy.get('#post_textbox').should('be.empty');
```

Comment:
This is related to Cypress - characters being typed on shortcut keypress.

Reply:
Aha! This makes sense. I was wondering why it was typing L and M :)